packages/uui-color-swatches/lib/uui-color-swatches.element.ts

```diff
@@ -137,7 +151,7 @@ export class UUIColorSwatchesElement extends LabelMixin('label', LitElement) {
     this.swatches.forEach(swatch => {
       swatch.selected = false;
       swatch.active = false;
-      swatch.selectable = true;
+      swatch.selectable = !swatch.disabled;
```

Comment:
When reset the selection the swatch should only be selectable if not disabled :)